src/main/kotlin/com/openlattice/shuttle/ShuttleServer.kt

```diff
@@ -1,35 +1,41 @@
 package com.openlattice.shuttle
 
 import com.kryptnostic.rhizome.configuration.websockets.BaseRhizomeServer
+import com.kryptnostic.rhizome.core.RhizomeApplicationServer.DEFAULT_PODS
+import com.kryptnostic.rhizome.pods.hazelcast.RegistryBasedHazelcastInstanceConfigurationPod
 import com.openlattice.auditing.pods.AuditingConfigurationPod
+import com.openlattice.auth0.Auth0Pod
 import com.openlattice.aws.AwsS3Pod
 import com.openlattice.datastore.pods.ByteBlobServicePod
 import com.openlattice.hazelcast.pods.MapstoresPod
 import com.openlattice.hazelcast.pods.SharedStreamSerializersPod
 import com.openlattice.jdbc.JdbcPod
 import com.openlattice.postgres.PostgresPod
-import com.openlattice.shuttle.pods.ShuttleMvcPod
-import com.openlattice.shuttle.pods.ShuttleServicesPod
-import com.openlattice.shuttle.pods.ShuttleServletsPod
+import com.openlattice.shuttle.pods.*
 import com.openlattice.tasks.pods.TaskSchedulerPod
 
 
 private val shuttlePods = arrayOf(
+        RegistryBasedHazelcastInstanceConfigurationPod::class.java,
+        JdbcPod::class.java,
         AuditingConfigurationPod::class.java,
         AwsS3Pod::class.java,
         ByteBlobServicePod::class.java,
-        JdbcPod::class.java,
         MapstoresPod::class.java,
         PostgresPod::class.java,
         SharedStreamSerializersPod::class.java,
+        ShuttleStreamSerializersPod::class.java,
         TaskSchedulerPod::class.java,
         ShuttleServicesPod::class.java,
         ShuttleServletsPod::class.java,
-        ShuttleMvcPod::class.java
+        ShuttleMapstoresPod::class.java,
+        Auth0Pod::class.java,
+        ShuttleSecurityPod::class.java,
+        MissionParametersPod::class.java
```

Comment:
Alphabetize please 😇

Reply:
Also I prefer to have these grouped into categories, similar to `Datastore.java` so that it's very easy to identify which pods are specific to Shuttle (which would be called O(shuttlePods)), which are infrastructural (which would be called O(rhizome/default)) and then webPods for any web-server/controller/api-surface pods

Reply:
I feel a certain voodoo that since this order works I should not change it, despite logic telling me otherwise. If I alphabetize and it breaks I'm blaming you lol.